### PR TITLE
Update to the interface version of the clipboard

### PR DESCRIPTION
**Pull request by erbridge**
Relies on limetext/backend#117.

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -1,5 +1,5 @@
-hash: aefc9f5f9214e4ff75f812de48b1fc9251ad8acbf3bf6a332f306a45704ff33b
-updated: 2016-08-12T21:00:49.083649551+01:00
+hash: 303c9cec0db7f8ba581f01afe8ff55e607b3bdf848daaeef3900e3940ef21d0a
```
Comment (thewhitetulip) [inline glide.lock:1]: @erbridge what is the significance of the `glide.lock` file?

Comment (erbridge) [inline glide.lock:1]: It pins dependencies, to make sure we're running the same versions of packages everywhere (or at least know what version we expect people to be running).

Comment (thewhitetulip) [inline glide.lock:1]: Which program generates the file?

Comment (erbridge) [inline glide.lock:1]: [Glide](https://github.com/Masterminds/glide)

**Comment by ricochet1k** [comment]
So you're just removing the clipboard?

**Comment by erbridge** [comment]
No. The system clipboard implementation will be set up by default in the backend, but can be replaced by something that implements the interface. I'm not sure why we were setting the clipboard functions in the frontend, anyway.

**Comment by ricochet1k** [comment]
Ah, okay. Looks good.

**Pull request merged**